src/query/procedure/module.cpp

```diff
@@ -325,15 +325,17 @@ bool SharedLibraryModule::Load(const std::filesystem::path &file_path) {
   dlerror();  // Clear any existing error.
   handle_ = dlopen(file_path.c_str(), RTLD_NOW | RTLD_LOCAL);
   if (!handle_) {
-    // NOLINTNEXTLINE(concurrency-mt-unsafe)
-    spdlog::error(utils::MessageWithLink("Unable to load module {}; {}.", file_path, dlerror(), "https://memgr.ph/modules"));
+    spdlog::error(utils::MessageWithLink("Unable to load module {}; {}.", file_path, dlerror(),
+                                         "https://memgr.ph/modules"));  // NOLINT(concurrency-mt-unsafe)
```

Comment:
```suggestion
    spdlog::error(utils::MessageWithLink("Unable to load module {}; {}.", file_path, dlerror(),
                                         "https://memgr.ph/modules"));
```

I think the other PR has to be merged to master before this PR can be merged, because that disables this check.